our-brothers-web/src/app/components/header/header.component.ts

```diff
@@ -1,16 +1,47 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, OnInit, OnDestroy } from '@angular/core';
 import { User } from 'firebase';
 import { AuthService } from 'src/app/services/auth.service';
 
+class HeaderSubMenus {
+  meetings: boolean;
+  about: boolean;
+}
+
 @Component({
   selector: 'app-header',
   templateUrl: './header.component.html',
   styleUrls: ['./header.component.scss']
 })
-export class HeaderComponent {
+export class HeaderComponent implements OnInit, OnDestroy {
+  isMobileView = false;
+  subMenusStates: HeaderSubMenus = {
+    meetings: false,
+    about: false
+  };
 
-  @Input()
-  public user: User;
+  @Input() public user: User;
 
   constructor(public authService: AuthService) { }
+
+  updateMobileViewState = () => this.isMobileView = window.innerWidth <= 1400;
```

Comment:
`@HostListener('window:resize', ['$event'])`. Then we will can rid off from OnInit and OnDestroy